examples/speaker_tasks/diarization/README.md

```diff
@@ -73,25 +73,48 @@ Some of important options in config file:
+You could also download *.nemo files from [this link](https://ngc.nvidia.com/catalog/models?orderBy=scoreDESC&pageNumber=0&query=SpeakerNet&quickFilter=&filters=) and specify the full path name to the speaker embedding model file (`*.nemo`).
 
- &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `diarizer.speaker_embeddings.model_path='path/to/titanet_large.nemo'` 
+`diarizer.speaker_embeddings.model_path='path/to/titanet_large.nemo'` 
  
 - **`diarizer.vad.model_path`: voice activity detection modle name or path to the model**
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  Specify the name of VAD model, then the script will download the model from NGC. Currently, we have 'vad_marblenet' and  'vad_telephony_marblenet' as options for VAD models.
+Specify the name of VAD model, then the script will download the model from NGC. Currently, we have 'vad_marblenet' and  'vad_telephony_marblenet' as options for VAD models.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `diarizer.vad.model_path='vad_telephony_marblenet'`
+`diarizer.vad.model_path='vad_telephony_marblenet'`
 
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  Instead, you can also download the model from [vad_marblenet](https://ngc.nvidia.com/catalog/models/nvidia:nemo:vad_marblenet) and [vad_telephony_marblenet](https://ngc.nvidia.com/catalog/models/nvidia:nemo:vad_telephony_marblenet) and specify the full path name to the model as below.
+Instead, you can also download the model from [vad_marblenet](https://ngc.nvidia.com/catalog/models/nvidia:nemo:vad_marblenet) and [vad_telephony_marblenet](https://ngc.nvidia.com/catalog/models/nvidia:nemo:vad_telephony_marblenet) and specify the full path name to the model as below.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `diarizer.vad.model_path='path/to/vad_telephony_marblenet.nemo'`
+`diarizer.vad.model_path='path/to/vad_telephony_marblenet.nemo'`
 
+- **`diarizer.speaker_embeddings.parameters.multiscale_weights`: multiscale diarization (Experimental)**
+
+Multiscale diarization system employs multiple scales at the same time to obtain a finer temporal resolution. To use multiscale feature, at least two scales and scale weights should be provided. The scales should be provided in descending order, from the longest scale to the base scale (the shortest). If multiple scales are provided, multiscale_weights must be provided in list format. The following example shows how multiscale parameters are specified and the recommended parameters.
+
+#### Example script: single-scale and multiscale
+Single-scale setting:
+```bash
+  python offline_diarization.py \
+     ... <other paramerters> ...
+     parameters.window_length_in_sec=1.5 \
+     parameters.shift_length_in_sec=0.75 \
+     parameters.multiscale_weights=null \
+```
+
+Multiscale setting (base scale - window_length 0.5 s and shift_length 0.25):
```

Comment:
just wondering is it necessary to introduce base scale concept?

Reply:
I think this is necessary because the actual output of the diarization result is based on the base scale. I think we should link paper in the NeMo docs.